### PR TITLE
Add stripes-authority-components

### PR DESCRIPTION
**Pull request by Dmytro-Melnyshyn**
Include `stripes-authority-components`.

### CODE REVIEW

**Comment by mkuklis** [comment]
@Dmytro-Melnyshyn I think the version of `stripes-authority-components` should be bumped to `1.0.0`. Anything below 1.0.0 (0.y.z) is considered to be not stable (not ready for public release) according to the semantic versioning:

https://semver.org/#spec-item-4

I think we also ran into some issues with versions below 1.0.0 in the past.

**Comment by Dmytro-Melnyshyn** [comment]
> @Dmytro-Melnyshyn I think the version of `stripes-authority-components` should be bumped to `1.0.0`. Anything below 1.0.0 (0.y.z) is considered to be not stable (not ready for public release) according to the semantic versioning:
> 
> https://semver.org/#spec-item-4
> 
> I think we also ran into some issues with versions below 1.0.0 in the past.

Thanks, @mkuklis, the version has been updated, please take a look.

**Review by funkymalc — APPROVED** [review]

**Comment by zburke** [comment]
@Dmytro-Melnyshyn, if we're providing `@folio/stripes-authority-components` at the platform-level (like we do `@folio/stripes` and `react` and a few other shared modules) then it must be moved to a peer-dependency at the UI module level rather than [included as a direct dependency as it is at present](https://github.com/folio-org/ui-plugin-find-authority/blob/d437188f21f501e63cddf7e0e15b431229dc3c9a/package.json#L66). 

In short, direct-deps at the platform-level should always be peer-deps at the application level. Direct-deps at the application level should not be present at the platform-level. Filed under [UIPFAUTH-9](https://issues.folio.org/browse/UIPFAUTH-9).
